OSSMDOC-578: Update Jaeger Operator default namespace. (#45632)
openshift/openshift-docs: modules/ossm-rn-new-features.adoc

This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs) and bug fixes.

> With this release, the {JaegerName} Operator is now installed to the `openshift-distributed-tracing` namespace by default.  Previously the default installation had been in the `openshift-operator` namespace.

Reviewer: ```suggestion
With this release, the {JaegerName} Operator is now installed to the `openshift-distributed-tracing` namespace by default.  Previously, the default installation was located in the `openshift-operator` namespace.
```
